app/assets/javascripts/miq_debug.js

```diff
@@ -71,7 +71,23 @@ $(function() {
   miq_bootstrap(el, 'miq.debug');
 });
 
-angular.module('miq.debug', [])
+function sanitize (data) {
```

Comment:
We should use a better name here now that it's a global.. `sanitizeError` maybe?